src/hearings/containers/request-hearing/hearing-judge/hearing-judge.component.ts

```diff
@@ -1,13 +1,13 @@
 import { Component, OnDestroy, OnInit, ViewChild } from '@angular/core';
-import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { FormArray, FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { ActivatedRoute } from '@angular/router';
 import { PersonRole } from '@hmcts/rpx-xui-common-lib/lib/models';
 import { Store } from '@ngrx/store';
-import { HearingJudgeNamesListComponent } from '../../../../hearings/components';
+import { HearingJudgeNameComponent, HearingJudgeNamesListComponent } from '../../../../hearings/components';
 import { Person } from '../../../../hearings/models/person.model';
 import { RefDataModel } from '../../../../hearings/models/refData.model';
 import { ValidatorsUtils } from '../../../../hearings/utils/validators.utils';
```

Comment:
seems it has a redudant ../,
../../../hearings/xxxx is the right path.

Reply:
Unable to get this one. Verified from my end and all looks good